### PR TITLE
Allow sections to be hidden from the Control Panel.

### PR DESCRIPTION
**Pull request by bloveless**
This will allow sections to be set as hidden from the sections array. The control panel theme will have to be updated to respect this flag.

### CODE REVIEW

**Comment by bloveless** [comment]
For a theme example on how see this PR https://github.com/pyrocms/accelerant-theme/pull/5

File: src/Ui/ControlPanel/Component/Section/Section.php [hunk 1]
```diff
@@ -435,6 +441,29 @@ public function setBreadcrumb($breadcrumb)
     }
 
     /**
+     * Get if this section is hidden.
+     *
+     * @return bool
+     */
+    public function getHidden()
```
Comment (RyanThompson) [inline src/Ui/ControlPanel/Component/Section/Section.php]: Can we do `isHidden` here ^_^

**Comment by RyanThompson** [comment]
How would you designate this section as being active then? No sections would be marked active?

**Comment by bloveless** [comment]
If there is a parent of a active child then it get a highlighted attribute. In this https://github.com/pyrocms/accelerant-theme/blob/4ebbc5730b2d67601a1726259bcf7cbea79960a1/resources/views/macros/menu.twig I'm also adding the class highlighted which would be applied to all the parent even if the child is hidden. I assume that is what you are asking about.

**Pull request closed without merging**